src/c++/perf_analyzer/test_perf_utils.cc

```diff
@@ -24,6 +24,9 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
```

Comment:
Were you planning on adding testing for `SerializeStringTensor()`, `SerializeExplicitTensor()`, or `ShapeTensorValuesToString()`?

Reply:
I didn't add tests for `SerializeStringTensor` and `SerializeExplicitTensor` because they were not immediately clear to me on how they behave. As for `ShapeTensorValuesToString`, I noticed the function was never used in the codebase so I was thinking if we should just delete it?

Reply:
Sorry for the late response. Makes sense. And good catch--feel free to remove `ShapeTensorValuesToString()` from the codebase.

Reply:
Addressing this at [#365 ](https://github.com/triton-inference-server/client/pull/365)